### PR TITLE
Feat/1056 brand updates, restyling (and a bonus bug fix)

### LINKED ISSUE #1056: Brand updates
- Update favicon
- Change to SIL Global

### PR DESCRIPTION
**Pull request by myieye**
Resolves #1056 

Rename to SIL Global and minor app logo/name touch ups:
image

Use (big) color icon for welcome message:
image
image

Fac icon:
image

Restyled app link to get rid of ugly underlines. After and before:
image

Password input field bug I fixed:
image

### CODE REVIEW

File: backend/FixFwData/FixFwData.csproj [hunk 1]
```diff
@@ -4,10 +4,10 @@
     <OutputType>WinExe</OutputType>
     <RootNamespace>FixFwData</RootNamespace>
     <Description>FixFwData</Description>
-    <Company>SIL International</Company>
-    <Authors>SIL International</Authors>
+    <Company>SIL Global</Company>
+    <Authors>SIL Global</Authors>
     <Product>LexBoxApi Testing</Product>
-    <Copyright>Copyright © 2023 SIL International</Copyright>
+    <Copyright>Copyright © 2024 SIL Global</Copyright>
```
Comment (hahn-kev) [inline backend/FixFwData/FixFwData.csproj:10]: we can actually just remove these lines that you edited, this project is just to fulfill a requirement and will never be published.

**Review by hahn-kev — DISMISSED** [review]
one small request and a question. But feel free to merge whenever you're ready.

File: frontend/src/lib/icons/AppLogo.svelte [hunk 2]
```diff
@@ -1,10 +1,21 @@
...
 
+{#if mono}
 <svg class={className} fill="currentColor">
   <!-- eslint-disable-next-line svelte/no-at-html-tags-->
   {@html logoMono}
 </svg>
+{:else}
+<picture class="my-0 {className}">
+  <source srcset={logoLight} media="(prefers-color-scheme: dark)">
+  <img src={logoDark} alt="Lexbox logo">
```
[the remark this answers is not in the document]
Comment (myieye) [reply frontend/src/lib/icons/AppLogo.svelte:19]: That's what I initially thought too. But, the light logo doesn't work well on a light background.
So, I initially thought that the light logo is "buggy", but then I realized that I think this is how they're intended to be used - it's how the designer presented them to us:
image

File: frontend/src/lib/layout/AppBar.svelte [hunk 3]
```diff
@@ -26,15 +26,15 @@
     </a>
   {/if}
   <div class="navbar justify-between bg-primary text-primary-content md:pl-3 md:pr-6">
-    <div class="flex">
-      <AppLogo class="h-[2.5em] w-[2.5em] mr-3"/>
-      <a href={loggedIn ? '/' : '/login'} class="flex flex-col text-lg md:text-3xl tracking-wider hover:underline">
-        {$t('appbar.app_name')}
-        <span class="text-xs md:text-[0.35em]/[2em]">
+    <a href={loggedIn ? '/' : '/login'} class="flex btn btn-primary text-left font-normal px-2">
```
[the remark this answers is not in the document]
Comment (myieye) [reply frontend/src/lib/layout/AppBar.svelte:29]: Yeah. This is what it looks like hovered:
image

**Pull request merged**